### PR TITLE
Add settings to change email subjects

### PR DESCRIPTION
**Pull request by adam-thomas**
@incuna/backend review please?

### CODE REVIEW

**Comment by adam-thomas** [comment]
OK.  So I have this test failure and I have no idea what the problem is.

There's one test (that I haven't touched) which is now not sending an email.  The test I added is an exact duplicate of the other test, up to the point where the email is/isn't sent, aside from `override_settings`.  I've tried removing `override_settings`, adding it to the original test, undoing the change in the notification function, and so on.  There are no tests in the same file with the same name as the original.  I HAVE NO IDEA WHAT'S GOING ON. D:

**Comment by adam-thomas** [comment]
Bump?  I need advice on this failure.

**Comment by adam-thomas** [comment]
halp?

**Comment by adam-thomas** [comment]
OK. If I delete the new test, the original test passes again.

**Comment by Ian-Foote** [comment]
What happens if you run the tests in [reverse order](https://docs.djangoproject.com/en/1.8/ref/django-admin/#django-admin-option---reverse)?

**Comment by adam-thomas** [comment]
I don't know if the `--reverse` is ignored by `run.py`, but the same failure occurs (???).

**Comment by adam-thomas** [comment]
I've also tried explicitly setting `email=something@unique.com` in each of the `UserFactory.create()` declarations to dodge clashing, and that didn't work either.

**Comment by adam-thomas** [comment]
This should fix it, thank you @Ian-Foote!

File: user_management/api/tests/test_views.py [hunk 1]
```diff
@@ -921,7 +935,7 @@ def test_delete(self):
         self.assertEqual(self.user, user)
 
 
-@patch(THROTTLE_RATE_PATH, new={'confirmations': '4/minute'})
+@patch(THROTTLE_RATE_PATH, new={'confirmations': '40/minute'})
```
Comment (Ian-Foote) [inline user_management/api/tests/test_views.py]: :trollface:

**Comment by adam-thomas** [comment]
hooray, Travis is going green now :D

File: user_management/api/tests/test_views.py (hunk 1 above)
Comment (adam-thomas) [inline user_management/api/tests/test_views.py]: FeelsBadMan

File: user_management/api/tests/test_views.py [hunk 2]
```diff
@@ -994,6 +1008,19 @@ def test_send_email_authenticated(self):
         expected = 'example.com account validate'
         self.assertEqual(email.subject, expected)
 
+    @override_settings(DUM_VALIDATE_EMAIL_SUBJECT='overridden subject')
+    def test_send_email_subject_setting(self):
+        """Assert the subject is affected by the DUM_VALIDATE_EMAIL_SUBJECT setting."""
+        user = UserFactory.create()
+        data = {'email': user.email}
+        request = self.create_request('post', auth=False, data=data)
```
Comment (Ian-Foote) [inline user_management/api/tests/test_views.py]: Maybe inline `data` here like you do in your other new test?

Comment (adam-thomas) [inline user_management/api/tests/test_views.py]: :+1:

**Comment by adam-thomas** [comment]
Updated

**Pull request merged**